compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/tiling.mlir

```diff
@@ -1763,3 +1763,75 @@ module attributes { transform.with_named_sequence } {
     transform.yield
   }
 }
+
+// -----
+
+#mapQ = affine_map<(batch, m, k1, k2, n) -> (batch, m, k1)>
+#mapK = affine_map<(batch, m, k1, k2, n) -> (batch, k2, k1)>
+#mapV = affine_map<(batch, m, k1, k2, n) -> (batch, k2, n)>
+#mapM = affine_map<(batch, m, k1, k2, n) -> (batch, m, k2)>
+#mapO = affine_map<(batch, m, k1, k2, n) -> (batch, m, n)>
+#mapR = affine_map<(batch, m, k1, k2, n) -> (batch, m)>
+
+func.func @online_attention(%query: tensor<192x1024x64xf32>, 
+                            %key: tensor<192x1024x64xf32>, 
+                            %value: tensor<192x1024x64xf32>, 
+                            %mask: tensor<192x1024x1024xf32>) 
```

Comment:
Can we get a test with the boolean mask?

Reply:
Added